react/src/pages/Pogoda_szuk.js

```diff
@@ -1,24 +1,59 @@
 import Navbar from '../components/Navbar';
 import Header from '../components/Header';
 import ButtonsPogoda from '../components/ButtonsPogoda';
+import Search from '../components/Weather/Search';
 import CurrentWeather from '../components/Weather/CurrentWeather';
+import { weatherApi } from '../components/Weather/Api';
+import { useState } from 'react';
+
 function PogodaSzukPage(){
```

Comment:
Tutaj również możemy podmienić nazwę na angielską: `SearchPage`, a dodatkowo przerobić to na funkcję strzałkową:
`function SearchPage() { ... }` => `const SearchPage = () => { ... }`